### DISCUSSION TITLE
live2d tests don't work

### DISCUSSION THREAD

**Discussion by 10100010010001001010** [axmolengine/axmol#1400] [Q&A] [answered]
I have the latest version of Axmol. Some files are missing, apparently there are errors in the cmake file. I transferred the extension files manually, but there is no Live2DCubismCore.h file at all. I can download it manually too, but as I understand, halx99 configures all extensions for axmol and nothing will work

**Comment by rh101** [comment]
The `Live2DCubismCore.h` is part of the runtime pack, and shouldn't be added manually.   It is part of the runtime zip file that is downloaded during the cmake project creation process.  the reason it is not working is because the Live2D extension is disabled for some reason, when it should be enabled for that test project.

Open the `tests/live2d-tests/CMakeLists.txt` file, and add this line to the top of the file:

`set(AX_ENABLE_EXT_LIVE2D ON CACHE BOOL "Build extension Live2D" FORCE)`

For example:
```
cmake_minimum_required(VERSION 3.10)

set(APP_NAME live2d_tests)

project(${APP_NAME})

set(AX_ENABLE_EXT_LIVE2D ON CACHE BOOL "Build extension Live2D" FORCE)

if(NOT DEFINED BUILD_ENGINE_DONE)
```

If you already have a build folder, then delete the `[build_folder]/CMakeCache.txt` file that is in there before you re-run cmake, otherwise the cached value of AX_ENABLE_EXT_LIVE2D won't be updated.

**Reply by rh101** [reply, marked as answer]
If you're using Axmol 2.x, then you will additionally need the changes in PR #1405 to get Live2D working.
